istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -673,8 +673,8 @@ func (o *RemoteSecretOptions) prepare(ctx cli.Context) error {
 	o.KubeOptions.prepare(ctx)
 
 	if o.ClusterName != "" {
-		if !labels.IsDNS1123Label(o.ClusterName) {
-			return fmt.Errorf("%v is not a valid DNS 1123 label", o.ClusterName)
+		if errs := utilvalidation.IsDNS1123Subdomain(o.ClusterName); len(errs) > 0 {
```

Comment:
K8s secret name can be DNS1123Subdomain, but I'm not 100% sure if the same cluster name format is valid in Istio since the it will be read in secretcontroller to reconcile multicluster things. @stevenctl could you please confirm this?